packages/block-library/src/video/edit.js

```diff
@@ -148,7 +156,7 @@ class VideoEdit extends Component {
 		} = this.props;
 		const { editing } = this.state;
 		const switchToEditing = () => {
```

Comment:
Should we rename the function as it now does not "switchToEditing" but toggles the editing?